lib/metadata_gen.js

```diff
@@ -1,14 +1,24 @@
 'use strict';
 
 const LinkParser = require('./links');
-
+/**
+ * @typedef {{reviewer: Collaborator}} Reviewer
+ */
```

Comment:
Is this from some specific typechecking/type annotation language? It might be nice to know for when I write my own PRs and I don’t recognize this as something I know :)

Reply:
It's just [JSDoc comments](http://usejsdoc.org/tags-typedef.html). I use vscode, it's supposed to understand this, but apparently it does not take `/// <reference path="./collaborators.js"/>` for non-typescript projects at the moment, so the intellisense stops at per-file scopes..but it's good enough for human to read as well, I normally annotate the code even not for the intellisense, this helps me remember what to put in fixtures and what to expect.

<img width="272" alt="screen shot 2017-10-29 at 1 57 48 am" src="https://user-images.githubusercontent.com/4299420/32137141-aba5aad4-bbdf-11e7-9761-10d18aeac2fd.png">